editors/sc-ide/core/sc_lexer.cpp

```diff
@@ -1,3 +1,4 @@
+
```

Comment:
@sonata-chen do you know whether this whitespace change added by clang-format? It's not that important, but we try to avoid accidental whitespace changes.

Reply:
I believe it's me accidentally added that empty line. I will forced-push again.